.test-infra/jenkins/CommonTestProperties.groovy

```diff
@@ -21,7 +21,10 @@ class CommonTestProperties {
         DATAFLOW("DataflowRunner", ":beam-runners-google-cloud-dataflow-java"),
         SPARK("SparkRunner", ":beam-runners-spark"),
         FLINK("TestFlinkRunner", ":beam-runners-flink_2.11"),
-        DIRECT("DirectRunner", ":beam-runners-direct-java")
+        DIRECT("DirectRunner", ":beam-runners-direct-java"),
+
+        DATAFLOW_PYTHON("DataflowRunner", "TestDataflowRunner"),
```

Comment:
We could divide an `SDK` enum and decide which dependency to choose based on it. This would have the advantage of keeping the original "Runner" semantics (no special cases for Python SDK). Seems like a low hanging fruit - do you think you could refactor this (of course if you agree with the approach)?

If for some reason this will be hard - let's do it in a separate PR

Reply:
Thanks! I added commit with a suggestion how might it work.